safekeeper/src/control_file_upgrade.rs

```diff
@@ -165,7 +165,7 @@ pub fn upgrade_control_file(buf: &[u8], version: u32) -> Result<SafeKeeperState>
             timeline_start_lsn: Lsn(0),
             local_start_lsn: Lsn(0),
             commit_lsn: oldstate.commit_lsn,
-            s3_wal_lsn: Lsn(0),
+            backup_lsn: Lsn(0),
```

Comment:
we are supposed to be multi-cloud, right? the code is not s3 specific, I think in this case backup_lsn is more appropriate name

Reply:
What are other non-s3 offerings for object storage? s3 looks like standard now.

Reply:
Google storage seems to be compatible with s3, Azure Blob Storage not. I don't mind either rename or leaving as is, but note that most probably cf file upgrade procedure should be fixed (or checked that it is ok) in case of rename.